FraihmworkHealthAndIntegrityApi.yaml

```
@@ -2191,6 +2429,74 @@ components:
           type: string
           description: The version of the component's configuration, if applicable
 
+    RecurringSchedule:
+      type: object
+      required:
+        - componentId
+        - schedules
+      properties:
+        componentId:
+          type: string
+          format: uuid
+          description: |
+            The unique identifier for the component. 
+        zone:
+          type: string
+          description: |
+            Specifies the time zone for the component. All schedules for this component will have this timezone.
```

Needs qualification. Is "Eastern Standard Time" acceptable? Probably not.

Should probably be a standard time zone id (UTC, GMT, EDT, EST, Z, ACWST, etc) and/or an hour offset (+/-01:30). Probably best to accept both
```suggestion
            Specifies the time zone for the component. All schedules for this component will have this time zone. Should be specified as either a standard time zone id (UTC, GMT, EST, etc.) or as an offset as seen in RFC3339 (+01:00, -02:30, etc.).```